webpack.config.js

```diff
@@ -74,12 +74,16 @@ module.exports = {
 				to: path.join(distRootPath, 'manifest.json'),
 				toType: 'file',
 			}
-		]),
+		],{
+			copyUnmodified: true // resolve conflict with `CleanWebpackPlugin`
+		}),
 		new webpack.DefinePlugin({
 			'NODE_ENV': JSON.stringify(nodeEnv),
 			'WEB_BROWSER': JSON.stringify(webBrowser),
 		}),
 		extensionReloader,
-		new CleanWebpackPlugin(),
+		new CleanWebpackPlugin({
+			// cleanStaleWebpackAssets: false, // another way to resolve conflict with `CopyWebpackPlugin`
```

Comment:
just to check - do we need both of those ( based on "another way" phrasing)?

Reply:
Just `copyUnmodified` should be enough. 

Both options were suggested on the thread I linked to, and they both worked for me, but I think `copyUnmodified` is the better solution. 

I left both there just in case. Maybe merge into another branch to test it then remove the commented one?

Reply:
I see. I'd prefer to go with just one of them if that's enough. Would appreciate if you can do the test you mention :). Otherwise - I'll take a look at it tonight